backend/src/useCases/uploads/uploadProcessing.ts

```diff
@@ -21,6 +21,7 @@ import {
   uploadsRepository,
 } from "../../repositories/uploads/uploads.js";
 import { filePartsRepository } from "../../repositories/uploads/fileParts.js";
+import { CID } from "multiformats";
```

Comment:
do we export this type in auto-sdk? if so we should export it, basically anything we need from multiformats should be exported in our package

Reply:
No, they're not exported I'll add an issue on the sdk